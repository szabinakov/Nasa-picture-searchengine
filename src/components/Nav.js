import React from 'react'
import {Link} from 'react-router-dom'
import '../styles/Nav.css'

const Nav = () => {
    return(
    <nav>
        <ul>
            <Link to='/'>
            <li className='Link'>Home</li>
            </Link>
            <Link to='/about'>
            <li className='Link'>About</li>
            </Link>
        </ul>
    </nav>
    )
}

export default Nav