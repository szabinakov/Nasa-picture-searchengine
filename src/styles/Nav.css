nav {
    display: flex;
    justify-content: space-around;
    align-items: center;
}
ul {
    width: 50%;
    display: flex;
    justify-content: space-around;
    align-items: center;
    list-style: none;
}

a {
    text-decoration: none;
    color: black;
}

a:hover {
    color: red;
}