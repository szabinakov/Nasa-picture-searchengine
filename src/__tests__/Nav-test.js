import React from 'react';
import { render } from '@testing-library/react';
import Nav from '../components/Nav';
import { BrowserRouter } from 'react-router-dom';

describe('Nav', () => {

    const {asFragment} = render (
    <BrowserRouter>
    <Nav/>
    </BrowserRouter>)

    it('it renders correctly', () => {
        
        expect(asFragment()).toMatchSnapshot()
    })
})