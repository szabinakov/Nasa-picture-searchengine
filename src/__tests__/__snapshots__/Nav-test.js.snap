// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav it renders correctly 1`] = `
<DocumentFragment>
  <nav>
    <ul>
      <a
        href="/"
      >
        <li
          class="Link"
        >
          Home
        </li>
      </a>
      <a
        href="/about"
      >
        <li
          class="Link"
        >
          About
        </li>
      </a>
    </ul>
  </nav>
</DocumentFragment>
`;
